api/src/main/java/info/rmapproject/api/auth/ApiUserServiceImpl.java

```diff
@@ -69,7 +69,7 @@ public AuthorizationPolicy getCurrentAuthPolicy() throws RMapApiException {
 	    if (authorizationPolicy == null) {
 	        throw new RMapApiException(ErrorCode.ER_COULD_NOT_RETRIEVE_AUTHPOLICY);
 	        }
-	    log.debug("Authorization policy retrieved with username " + ((authorizationPolicy.getUserName()==null) ? "" : authorizationPolicy.getUserName()));
+	    LOG.debug("Authorization policy retrieved with username {}", ((authorizationPolicy.getUserName()==null) ? "" : authorizationPolicy.getUserName()));
```

Comment:
So it should be ok to not do a `null` check here, for a couple of reasons:
- SLF4J loggers can output a `null` reference without throwing an NPE
- If the semantics of `null` and the empty string are different, reporting the empty string may be misleading, especially to a developer debugging an NPE

Obviously, if there were a possibility of `authorizationPolicy` being `null`, then you'd need to perform a `null` check before calling `authorizationPolicy.getUserName()`.

Reply:
good to know!